ironmon_tracker/constants/MoveData.lua

```diff
@@ -4050,7 +4050,7 @@ MoveData.MOVES_MASTER_LIST = {
 		pp = "10",
 		accuracy = "100",
 		category = MoveData.MOVE_CATEGORIES.STATUS,
-		description = "Changes the target's ability to insomnia."
+		description = "Changes the target's ability to insomnia for as long as it remains on the field."
```

Comment:
I missed this in the initial description write up, but insomnia and other abilities should be capitalized like move names (for clarity and helping them stand out)